<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tasistencia</class>
 <widget class="QMainWindow" name="tasistencia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
border: 3px solid rgb(0, 0, 127);
background-color: rgb(170, 170, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QLabel{
border: none;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QComboBox{
border: none;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}
QLineEdit{
border: none;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;}

QPushButton{
border: 1px solid rgb(0, 0, 127);
border-radius:10px;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(170, 85, 255);
}

QPushButton:hover{
background-color: rgb(170, 255, 255);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
border: none;
	background-color: rgb(158, 158, 158);
}</string>
         </property>
         <layout class="QHBoxLayout" name="layout_principal">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_title">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>LLAMAR ASISTENCIA</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ampliar">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: rgb(0, 0, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border:none;
border-radius:0px;
}

QPushButton:hover{
	background-color: rgb(170, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>â– </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_close">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border:none;
border-radius:0px;
}

QPushButton:hover{
	background-color: rgb(170, 0, 255);
}</string>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="3">
            <widget class="QPushButton" name="btn_delete">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:none;
border-radius:0px;
}</string>
             </property>
             <property name="text">
              <string>Eliminar</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="change_code_project">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(85, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Hello</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="change_name_project">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(85, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QDateEdit" name="dt_fecha_list">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 135, 203);
border: 1px solid rgb(0, 0, 127);
border-radius:0px;</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="date">
              <date>
               <year>2023</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_37">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Responsable:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_33">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Nombre del proyecto:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="lineEdit_delete_dni">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
             <property name="placeholderText">
              <string>Ingrese el DNI</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Codigo de proyecto:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="change_responsible_name">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(85, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_36">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Fecha:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="btn_add_new">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:none;
border-radius:0px;
}</string>
             </property>
             <property name="text">
              <string>Agregar nuevo</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="lineEdit_dni_add">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
             <property name="placeholderText">
              <string>Ingrese el DNI</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QTableWidget" name="table_asistencia">
           <property name="styleSheet">
            <string notr="true">border: 1px solid rgb(0, 0, 127);
border-radius:1px;
background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="autoScrollMargin">
            <number>16</number>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Asistencia</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>DNI</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nombres</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Apellidos</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ObservaciÃ³n</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>JustificaciÃ³n</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_guardar_asistencia">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Guardar lista</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
